dagshub/data_engine/client/loaders/base.py

```diff
@@ -23,13 +23,19 @@ def __init__(
         for_dataloader: bool = False,
     ):
         """
-        query_result: <dagshub.data_engine.client.models.QueryResult>
-        metadata_columns: columns that are returned from the metadata as part of the dataloader
-        file_columns: columns with a datapoint metadata that are files
-        strategy: preload|background|lazy; default: lazy
-        tensorizers: auto|image|<function>
-        savedir: location at which the dataset is stored
-        processes: number of parallel processes that download the dataset
+        Initialize a dataset using the specified parameters.
+
+        Args:
+    
+        query_result (<dagshub.data_engine.client.models.QueryResult>): The query result containing dataset entries.
```

Comment:
Why use the full notation here and the Object notation in the previous files? We should be consistent. IMO either have it as `QueryResult` here, or change previous usages to `<dagshub.data_engine.client.models.QueryResult>` unless there's a specific reason to change it here?

Reply:
Multiple libraries have their own QueryResult, thats why this specification was provided.